### DISCUSSION TITLE
what is the  meaning of “Effect of a Variable”？

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#148] [Q&A] [answered]
what is the  meaning of “Effect of a Variable”？
![image](https://user-images.githubusercontent.com/5112397/148149879-a7bac029-a580-48e6-b9a7-4a8902afedd1.png)

library(randomForestSRC)
airq.obj <- rfsrc(Ozone ~ ., data = airquality)

partial.obj <- partial(airq.obj,
                       partial.xvar = "Wind",
                       partial.values = airq.obj$xvar$Wind)
pdta <- get.partial.plot.data(partial.obj)
plot(pdta$x, pdta$yhat, type = "b", pch = 16,
     xlab = "wind", ylab = "partial effect of wind")

![image](https://user-images.githubusercontent.com/5112397/148150019-ba71ea43-2b00-400d-8328-cd8f3ca7770e.png)
and what is the meaning of this curve？

**Comment by ishwaran** [comment, marked as answer]
We haven't completed the partial plot vignette, but you can take a look at the following temporary vignette which explains what a partial plot is and gives further references to review.

https://luminwin.github.io/randomForestSRC/articles/partial.html#run-functions-and-save-plots-1
